client/wildcard/src/components/NavBar/NavItem.module.scss

```
@@ -1,5 +1,32 @@
 @import 'branded/src/global-styles/breakpoints.scss';
 
+.item {
+    display: flex;
+    align-items: stretch;
+    margin: 0 0.75rem;
+    &:first-child {
+        margin-left: 0;
+    }
+    &:last-child {
+        margin-right: 0;
+    }
+    @include media-breakpoint-down(sm) {
```

```suggestion
@import 'wildcard/src/global-styles/breakpoints.scss';

.item {
    display: flex;
    align-items: stretch;
    margin: 0 0.75rem;
    &:first-child {
        margin-left: 0;
    }
    &:last-child {
        margin-right: 0;
    }
    @media (--md-breakpoint-down) {
```

You should be able to update these to use our new custom media queries now!